crates/cli/Cargo.toml

```diff
@@ -13,12 +13,11 @@ nix = { version = "0.29.0", features = ["process"] }
 serde_json = "1.0.135"
 libpkgx = { path = "../lib" }
 
-[target.'cfg(not(target_os = "macos"))'.dependencies]
-native-tls = { version = "0.2", features = ["vendored"] }
-# ^^ we want to be standalone and just work on minimal images
-
 [target.'cfg(not(target_os = "macos"))'.dependencies]
 rusqlite = { version = "0.32.1", features = ["bundled"] }
+native-tls = { version = "0.2", features = ["vendored"] }
+# ^^ this is a transitive dependency
+# ^^ we vendor OpenSSL ∵ we want to be standalone and just work inside minimal docker images
```

Comment:
if you need static `ffi`, consider: https://github.com/pkgxdev/pantry/commit/b6e6bd6744cd0c1061f97b0592428ebc59bbd53e